integration/test-integration/src/test/resources/dockerCompose/cc-cacert-mounted-mtls.yaml

```diff
@@ -73,7 +73,7 @@ services:
       - ENFORCER_REGION=UNKNOWN
       - XDS_MAX_MSG_SIZE=4194304
       - XDS_MAX_RETRIES=3
-      - JAVA_OPTS=${JAVA_OPTS}
+      - JAVA_OPTS=${JAVA_OPTS} -javaagent:/home/wso2/lib/org.jacoco.agent-0.8.8-runtime.jar=destfile=/home/wso2/lib/dropins/aggregate.exec,append=true
```

Comment:
Please don't commit this `-javaagent:/home/wso2/lib/org.jacoco.agent-0.8.8-runtime.jar=destfile=/home/wso2/lib/dropins/aggregate.exec,append=true` it's used only for the codecov. Remove in other places (dockerfiles in this PR) as well.